### DISCUSSION TITLE
Current iteration

### DISCUSSION THREAD

**Discussion by RobTF** [dotnet/maui#15600] [General] [answered]
Hi,

The current "[What's the team currently working on?](https://github.com/orgs/dotnet/projects/194/views/16)" link goes to to the current iteration, but there are no items on the iteration. The last iteration was iteration 6 which ended on Jun 1.

Is this still being kept up to date?

thanks

**Comment by hartez** [comment, marked as answer]
My fault, that list has been updated. Thanks for pointing it out!

**Reply by RobTF** [reply]
Great, thanks!
